.idea/runConfigurations/lexos_core.xml

```diff
@@ -6,9 +6,9 @@
     <envs>
       <env name="PYTHONUNBUFFERED" value="1" />
     </envs>
-    <option name="SDK_HOME" value="" />
+    <option name="SDK_HOME" value="$PROJECT_DIR$/../anaconda3/bin/python" />
```

Comment:
I think you want to use module sdk here

Reply:
I changed the interpreter settings to project default, I hope it will do the trick.